1
Traceback (most recent call last):
  File "test.py", line 3, in <module>
    1/0
ZeroDivisionError: division by zero
